app/src/main/java/org/fossasia/openevent/general/search/SearchLocationActivity.kt

```diff
@@ -25,6 +46,60 @@ class SearchLocationActivity : AppCompatActivity() {
         this.supportActionBar?.title = ""
         val bundle = intent.extras
         var fromSearchFragment = false
+        locationProgressBar.visibility = View.GONE
+        if(BuildConfig.FLAVOR.equals("playStore")){
```

Comment:
This is not what I meant

Reply:
What exactly I need to do @iamareebjamal ?

Reply:
There will a component called `GeoLocationUI` with a method `configure` 

Do everything in that class and simply call `configure` in the Activity. In FDroid code, `configure` function will do nothing